sdk/python/tests/test_historical_retrieval.py

```diff
@@ -557,3 +573,22 @@ def test_historical_features_from_bigquery_sources(
             .reset_index(drop=True),
             check_dtype=False,
         )
+
+
+@pytest.mark.integration
+def test_timestamp_bound_inference_from_entity_df_using_bigquery():
+    start_date = datetime.now().replace(microsecond=0, second=0, minute=0)
+    (_, _, _, entity_df, start_date) = generate_entities(
+        start_date, infer_event_timestamp_col=True
+    )
+
+    client = bigquery.Client()
+    table_id = "foo.table_id"
+    table = upload_entity_df_into_bigquery(client, entity_df, table_id)
```

Comment:
Any reason we can't reuse https://github.com/feast-dev/feast/blob/e8a361fc0a8994d846aeb3d8ae8e74db79003bc9/sdk/python/tests/test_historical_retrieval.py#L226:5?

Reply:
Actually, I think a little refactoring and we can use it just fine. Just pushed that change!

Reply:
@woop Unless there are other changes you would like me to make to the test, I think this should be good to ship? Again, happy to make last minute edits if you see any are necessary!

Reply:
Update: Changed typo in assert statement (use `==` now instead of `is`) after seeing the integration test had failed below, now all integration tests are passing.